doc/instances.md

```diff
@@ -670,6 +671,22 @@ security.gid    | int       | 0             | no        | What GID to drop privi
 lxc config device add <instance> <device-name> proxy listen=<type>:<addr>:<port>[-<port>][,<port>] connect=<type>:<addr>:<port> bind=<host/instance>
 ```
 
+### Type: unix-hotplug
+Unix hotplug device entries make the requested unix device
+appear in the container's `/dev` and allow read/write operations to it
+if the device exists on the host system.
+
+The following properties exist:
+
+Key         | Type      | Default           | Required  | Description
+:--         | :--       | :--               | :--       | :--
+vendorid    | string    | -                 | no        | The vendor id of the unix device
+productid   | string    | -                 | no        | The product id of the unix device
+uid         | int       | 0                 | no        | UID of the device owner in the container
```

Comment:
@Lilyjjo we are introducing other types of instance types rather than just containers, so to future proof this new doc please can you replace "container" with "instance" in this section. Thanks